src/peft/tuners/lora/config.py

```diff
@@ -225,5 +249,5 @@ def __post_init__(self):
                 raise ValueError("`loftq_config` must be specified when `init_lora_weights` is 'loftq'.")
 
         # convert loftq_config to dict
-        if self.loftq_config is not None and not isinstance(self.loftq_config, dict):
+        if self.loftq_config and not isinstance(self.loftq_config, dict):
```

Comment:
Just flagging out this change in case it is not intended but I think it is perfectly ok to have it

Reply:
This is because `loftq_config` is type `Union[LoftQConfig, dict]` (by default an empty dict), so `None` is not an option. But even if a user passes `None`, the logic would still be sound, so I made this change.